### ISSUE TITLE
Notifier does not resolve on Ubuntu

### ISSUE THREAD

**Issue by jrseliga** [jolicode/JoliNotif#37]
### System
> *OS*: Ubuntu 17.04
> *PHP*: 7.0
> *JoliNotif*: 1.1.1

### Details
Notifications do not display

### Troubleshooting
If I force `$bestNotifier` to be `NotifySendNotifier` (line 37 in `Joli\JoliNotif\NotifierFactory`

```
$bestNotifier = new \Joli\JolieNotif\Notifier\NotifySendNotifier;
```
I do receive a desktop notification

### Issue
`isSupported()` is incorrectly returning `false` on `NotifySendNotifier`

**Comment by lyrixx** [comment]
Hello @jrseliga.

Thanks for your bug report. It's very detailed and this is much appreciated.

The issue is command from symfony/process. There is a regression in this lib. I created [an issue](https://github.com/symfony/symfony/issues/23495) for that.

In the meantime, you can lock symfony/process to 3.2.* in your project to make it work.

**Comment by lyrixx** [comment]
Hello. We have merged a PR that should solve your issue.

Thanks again for the report.

**Comment by pyrech** [comment]
FYI the fix is available in the 1.1.2 release :wink:
